modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -177,8 +179,15 @@ private void runInitScriptIfRequired(final ConnectionUrl connectionUrl, Database
         if (connectionUrl.getInitScriptPath().isPresent()) {
             String initScriptPath = connectionUrl.getInitScriptPath().get();
             try {
-                URL resource = Thread.currentThread().getContextClassLoader().getResource(initScriptPath);
-
+                URL resource;
+                if (initScriptPath.startsWith(FILE_PATH_PREFIX)) {
+                    //relative project path
+                    String relativePath = System.getProperty("user.dir") + "/" + initScriptPath.substring(FILE_PATH_PREFIX.length());
```

Comment:
why are you using `user.dir` here?
Also, why not just `return new URL(initScriptPath)`? :)

Reply:
use `user.dir` Use the project path